MovieTime/MovieTime/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
```

Comment:
general question: what is the overall architecture you have adopted for this app (for example, MVC) and why did you choose it?

Reply:
The **MVVM** architecture was used in the Project.
The main reasons for using this architecture are the separation of concepts (business logic vs. user interface) and testability.
The *ViewModel* helps the data translation into something the view can use, and in processing / implementing some business rules.
In addition, because of the difficulty of testing *ViewControllers* (great relationship to the view layer). Using the *ViewModel* for data manipulation helps a lot when creating the tests and decoupling both parts, since the* ViewModel* should never have a reference to the *ViewController*.The **MVVM architecture** was used in the Project.
The main reasons for using this architecture are the separation of concepts (business logic vs. user interface) and testability.
The *ViewModel* helps the data translation into something the view can use, and in processing / implementing some business rules.
In addition, because of the difficulty of testing ViewControllers (great relationship to the view layer). Using the *ViewModel* for data manipulation helps a lot when creating the tests and decoupling both parts, since the *ViewModel* should never have a reference to the *ViewController*.